frontend/views/pages/GroupSettings.vue

```diff
@@ -1,147 +1,199 @@
 <template lang='pug'>
-page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
+page.c-page
   template(#title='') {{ L('Group Settings') }}
   template(#description='') {{ L('Changes to these settings will be visible to all group members') }}
 
-  page-section(title='')
+  avatar-upload(variant='group')
 
-    form
-      .field
-        i18n.label(tag='label') Group name
-
-        //- BUG: TODO: cannot bind directly to :value here, as any changes
-        //- to groupSettings must be done via proposals and/or
-        //- like it's done in UserProfile.vue
-        input.input.is-large.is-primary(
-          ref='name'
+  page-section
+    form(@submit.prevent='saveSettings')
+      label.field
+        i18n.label Group name
+        input.input(
           type='text'
-          name='groupName'
-          :class='{ error: $v.form.groupName.$error }'
-          :value='groupSettings.groupName'
-          @input='update'
-          @keyup.enter='next'
+          :class='{error: $v.form.groupName.$error}'
+          v-model='form.groupName'
+          @input='debounceField("groupName")'
+          @blur='updateField("groupName")'
+          v-error:groupName=''
```

Comment:
cc @pieer and @taoeffect 

Note: These 5 lines are needed in every input that needs validation besides "required". So far we have it in >10 places...

What do you think of creating a directive that handles it for us? What name do you suggest?
- `v-form:groupName=''`
- `v-model-debounce:groupName=''`
- `v-validate:groupName=''`
- _what else?_

If you agree I can create an issue for that.

Reply:
I think, since it's so specific to what we're doing, a name like `v-gi-form:groupName=''` or something similar would make sense. Minor issue, but yeah, feel free to create an issue if you'd like

Reply:
issue created #819